### PR TITLE
Tiany1 pan ansys 1

### PR DESCRIPTION
**Pull request by Tiany1Pan**

### CODE REVIEW

File: book/software/applications/ansys.md [hunk 1]
```diff
@@ -2,14 +2,15 @@
...
 ## Setting up the license
 
 The license is obtained by setting the `ANSYSLMD_LICENSE_FILE` variable. To set it in bash do:
 
 ```bash
 $ export ANSYSLMD_LICENSE_FILE=<port>@<host>
+$ export LSTC_LICENSE=ANSYS
 ```
 
 If accessing for the first time, there is an additional step (see the next section on this page) required to choose whether you are using commercial or academic licenses. To get the values of `<port>@<host>` specific to your group/department please contact the Client IT Team via [Service Now](https://it.leeds.ac.uk/it).
```
Comment (jhodrien) [inline book/software/applications/ansys.md:16]:
```suggestion
To get the values of `<port>@<host>` specific to your group/department please contact the Client IT Team via [Service Now](https://it.leeds.ac.uk/it).
```

File: book/software/applications/ansys/chemkin.md [hunk 2]
```diff
@@ -13,7 +13,7 @@ More information about Chemkin can be found on the [Ansys Website](https://www.a
 To use Chemkin, you must have connected to ARC using [X graphical forwarding](../../../getting_started/logon/x11-graphics)), loaded ansys/19.2 and assigned licence server information.
 
 ```{note}
-More details about using Ansys for the first time and setting license server details can be found on the [Ansys page](ansys:additional-steps).
+More details about using Ansys for the first time and setting license server details can be found in the License Set-up for Legacy Ansys Modules section..
```
Comment (jhodrien) [inline book/software/applications/ansys/chemkin.md:16]:
```suggestion
More details about using the old versions of Ansys for the first time and setting license server details can be found in the License Set-up for Legacy Ansys Modules section.
```

File: book/software/applications/ansys/lsdyna.md [hunk 3]
```diff
@@ -9,65 +9,65 @@ More information about LS-DYNA can be found on the [Ansys Website](https://www.a
 ## Running LS-DYNA
 
 ```{note}
-More details about using Ansys for the first time and setting license server details can be found on the [Ansys page](ansys:additional-steps).
+More details about using Ansys for the first time and setting license server details can be found in the License Set-up for Legacy Ansys Modules section..
```
Comment (jhodrien) [inline book/software/applications/ansys/lsdyna.md:12]:
```suggestion
More details about using the old versions of Ansys for the first time and setting license server details can be found in the License Set-up for Legacy Ansys Modules section.
```

File: book/software/applications/ansys/legacy.md [hunk 4]
```diff
@@ -0,0 +1,47 @@
+# Additional steps for using legacy Ansys modules the first time
+
+Before running the old versions of Ansys module (older than 19.2) for the first time, there is a per-user setting that chooses whether commercial (default) or academic licenses should be used.
+
+**For research purposes you should normally be using the academic licenses, unless you are untertaking commercially funded research.**
+
+**Check with your supervisor if you need clarification on this.**
+
+You will need a SSH connection with X-forwarding enabled to configure this setting. For more information on how to connect to the ARC systems and to enable X-forwarding visit the [Logging on page](../../getting_started/logon/x11-graphics).
```
Comment (jhodrien) [inline book/software/applications/ansys/legacy.md:9]:
```suggestion
You will need a SSH connection with X-forwarding enabled to configure this setting. For more information on how to connect to the ARC systems and to enable X-forwarding visit the [Logging on page](../../../getting_started/logon/x11-graphics).
```

**Review by jhodrien — CHANGES_REQUESTED** [review]
Looks sound.

You also need to update _toc.yml to include the legacy page you've introduced.

**Comment by Tiany1Pan** [comment]
Hi @jhodrien, many thanks for the comments, I've updated _toc.yml, please have a look.

**Review by jhodrien — APPROVED** [review]

**Pull request merged**